File: agilemeets_mobile/lib/data/models/profile/profile_dto.dart
class ProfileDTO {
  final String? fullName;
  final String? email;
  final String? countryName;
  final DateTime? birthDate;
  final String? organizationName;

  ProfileDTO({
    this.fullName,
    this.email,
    this.countryName,
    this.birthDate,
    this.organizationName,
  });

  factory ProfileDTO.fromJson(Map<String, dynamic> json) {
    return ProfileDTO(
      fullName: json['fullName'],
      email: json['email'],
      countryName: json['countryName'],
      birthDate: json['birthDate'] != null 
        ? DateTime.parse(json['birthDate']) 
        : null,
      organizationName: json['organizationName'],
    );
  }
} 

File: Backend/BaseMgtService/Senior.AgileAI.BaseMgt.Application/DTOs/ProfileDTO.cs
namespace Senior.AgileAI.BaseMgt.Application.DTOs
{
    public class ProfileDTO
    {
        public required string FullName { get; set; }
        public required string Email { get; set; }
        public required string CountryName { get; set; }
        //TODO: public string ProfilePicture { get; set; }
        public required DateOnly BirthDate { get; set; }
        public string? OrganizationName { get; set; }

    }
}